enterprise/models.py

```diff
@@ -1651,13 +1651,17 @@ def is_active(self):
         """
         Returns True iff this enrollment is currently active.
         """
+        if not self.course_enrollment:
```

Comment:
👍

Reply:
yeah the tests were not passing without this anyway